NDTensors/src/lib/Sectors/src/Sectors.jl

```diff
@@ -0,0 +1,19 @@
+module Sectors
+
+using ..SortedSets
+import ..SortedSets: AbstractSmallSet
+using InlineStrings
+using HalfIntegers
+
+import LinearAlgebra: norm
+import Base: *, ==, getindex, intersect, union, setdiff, symdiff, iterate, length, show
```

Comment:
Recently we have been using the style of explicitly listing functions that are being used in `using` statements, such as:
```julia
using InlineStrings: String7
```
Additionally, we have been listing those at the tops of the files where those functions are actually being used, instead of in the top level `module` definition.

Also, instead of `import Base: getindex`, we have been moving towards the style of overloading with `Base.getindex`.

I think it makes thing much more explicit, i.e. you can read the file locally and see where the functions were initially defined.

Reply:
Great